lib/v2/screens/app/interactor/viewmodels/app_bloc.dart

```diff
@@ -71,4 +57,30 @@ class AppBloc extends Bloc<AppEvent, AppState> {
     _shouldShowCancelGuardianAlertMessage.cancel();
     return super.close();
   }
+
+  void initDynamicLinks() async {
+    FirebaseDynamicLinks.instance.onLink(onError: (error) async {
+      print('onLinkError');
+      print(error.message);
+    }, onSuccess: (dynamicLink) async {
+      final Uri? deepLink = dynamicLink?.link;
+
+      if (deepLink != null) {
+        print('onSuccess');
+        print(deepLink.toString());
+        print(deepLink.data.toString());
+        add(HandleIncomingFirebaseDeepLink(deepLink));
+      }
+    });
+
+    final PendingDynamicLinkData? data = await FirebaseDynamicLinks.instance.getInitialLink();
+    final Uri? deepLink = data?.link;
+
+    if (deepLink != null) {
+      print('PendingDynamicLinkData');
+      print(deepLink.data.toString());
+      print(deepLink.toString());
```

Comment:
Debug prints? You can use the event to display values too. (not sensitive data of curse).

Reply:
yes